pkg/client/session.go

```diff
@@ -105,3 +109,11 @@ func (c *immuClient) CloseSession(ctx context.Context) error {
 func (c *immuClient) GetSessionID() string {
 	return c.SessionID
 }
+
+func (c *immuClient) keepAliveErrorHandler(sessionID string, err error) {
+	if err == sessions.ErrSessionNotFound || err == ErrNotConnected || strings.Contains(err.Error(), "session not found") {
+		c.Logger.Errorf("heartbeater KeepAlive error, sessionID %s error %v", sessionID, err)
+
+		panic(fmt.Errorf("heartbeater KeepAlive error, sessionID %s error %v", sessionID, err))
```

Comment:
This is just for us to test this situation right? Because I don't think we want to have a panic in the client.

Reply:
the intention is to grab info about the immudb connection state

Reply:
it shouldn't panic... in case it's desired such error handler could be externally specified

Reply:
resolved